test/integration/options.spec.js

```diff
@@ -180,4 +183,28 @@ describe('options', function () {
       });
     });
   });
+
+  describe('--help', function () {
+    it('works despite the presence of mocha.opts', function (done) {
+      var output = '';
+      try {
+        fs.mkdirSync(path.resolve(__dirname, 'test-env'));
+        fs.mkdirSync(path.resolve(__dirname, 'test-env/test'));
+        fs.writeFileSync(path.resolve(__dirname, 'test-env/test/mocha.opts'), 'foo');
+        var mochaLoc = path.resolve(__dirname, '../../bin/mocha');
+        output = '' + childProcess.execSync(mochaLoc + ' -h', {cwd: path.resolve(__dirname, 'test-env')});
+      } catch (e) {}
+      try {
+        fs.unlinkSync(path.resolve(__dirname, 'test-env/test/mocha.opts'));
+      } catch (e) {}
+      try {
+        fs.rmdirSync(path.resolve(__dirname, 'test-env/test'));
+      } catch (e) {}
+      try {
+        fs.rmdirSync(path.resolve(__dirname, 'test-env'));
+      } catch (e) {}
+      assert(output.indexOf('Usage:') >= 0);
+      done();
```

Comment:
The entire test is synchronous, and as such, the `done` callback is not needed here.